README.md

```diff
@@ -54,7 +54,7 @@ from pycape import Cape
 from pycape import FunctionRef
 
 client = Cape(url="wss://enclave.capeprivacy.com")
-function_id = "X9rgtUVm6yYrPMZamBWTZY"
+function_id = "2heeV48kLCAsvj6nYY87Fh"
```

Comment:
Maybe these should be placeholders

Reply:
I think these ones make sense so they can just run them without deploying a function?